app/views/users/profile.html.erb

```diff
@@ -105,28 +105,37 @@
 
   <div id="social-links">
 
-    <div class="profile_icons d-flex justify-content-center" style="gap: 8px">
-      <% if @twitter.nil? == false %>
-        <a href="<%= @twitter %>" >
-          <i class="fa fa-twitter fa-3x" aria-hidden="true"></i>
-        </a>
-      <% end %>
-      <% if @facebook.nil? == false %>
-        <a href="<%= @facebook %>" >
-          <i class="fa fa-facebook fa-3x" aria-hidden="true"></i>
-        </a>
-      <% end %>
-      <% if @github.nil? == false %>
-        <a href="<%= @github %>" >
-          <i class="fa fa-github fa-3x" aria-hidden="true"></i>
-        </a>
-      <% end %>
-      <% if @instagram.nil? == false %>
-        <a href="<%= @instagram %>" >
-          <i class="fa fa-instagram fa-3x" aria-hidden="true"></i>
-        </a>
-      <% end %>
-    </div>
+        <div class="profile_icons d-flex justify-content-center" style="gap: 8px">
+          <% if @twitter_user.nil? == false %>
+          <span>
+           <a aria-label="Delete tag" data-confirm="Are you sure you want to delete it?" class="tag-delete" data-remote="true" href="/profile/tags/delete/<%= @profile_user.id %>?name=<%= @twitter_user %>" data-method="delete">
+            <span class="badge badge-secondary delete_badge mb-1">x</span></a>         
+            <a href="<%= @twitter %>"><i class="fa fa-twitter fa-3x"></i></a>
```

Comment:
`@twitter` won't work now since we've removed those variables. It's now time for you to use the `social_link` helper.
```suggestion
            <a href="<%= social_link('twitter', @twitter_user)%>"><i class="fa fa-twitter fa-3x"></i></a>
```

Reply:
Also, can you indent the `div` element properly? Thanks!

Reply:
Hi @Forchapeatl, can you also make the changes I suggested above? Thank you!

Reply:
@TildaDares  , I'm sorry for the late reply I didn't see this message. Sorry , I will work on it right now